### DISCUSSION TITLE
[Support]: Is there a way to detect if a stream or snapshot is down in go2rtc rest interface

### DISCUSSION THREAD

**Discussion by sundansx** [blakeblackshear/frigate#10118] [General Support] [answered]
### Describe the problem you are having

I want to run a python stats script using the go2rtc rest interface availabe at the url "http://<hostname>:5000/api/go2rtc/streams by parsing the json and processing it.  I am looking for something similar to what is in the go2rtc web interrface as a 1 or 0 to indicate the stream state.  Does this exist in the current interface?

### Version

0.13.2-6476f8a

### Frigate config file

```yaml
not important
```

### Relevant log output

```shell
not important
```

### FFprobe output from your camera

```shell
not important
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

not important

**Comment by moandcompany** [comment, marked as answer]
There's a Frigate REST API endpoint for cameras at ```GET /api/stats``` that can be used to accomplish something similar using the `camera_fps` attribute (i.e. a value of zero (0) would suggest a problem with the camera or input stream).

https://docs.frigate.video/integrations/api/#get-apistats

**Reply by NickM-27** [reply]
this information is also exposed via mqtt (and the home assistant integration)

**Comment by sundansx** [comment]
great, thanks.  I see it on the rest api and I just pulled up MQTT explorer and see it is being sent to my Mosquito instance as well.  The MQTT stuff might be the way to go.  I can massage it with NodeRed into domoticz messages and pull it into a device.  I don't use HA, just frigate.
thanks again for a very quick answer.
